src/benchmarks/gc/GC.Infrastructure/Notebooks/Reports.dib

```diff
@@ -588,21 +588,169 @@ public static class Metrics
+        },
+        "CurrentGCIndex", 
+        "#");
+
+        public static Metric<TraceGC> TotalSOHStableSize = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.TotalSOHStableSize;
+        },
+        "TotalSOHStableSize", 
+        "bytes");
+
+        public static Metric<TraceGC> MedianThroughputCostPercent = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.MedianThroughputCostPercent;
+        },
+        "MedianThroughputCostPercent", 
+        "%");
+
+        public static Metric<TraceGC> CurrentAroundTargetAccumulation = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.CurrentAroundTargetAccumulation;
+        },
+        "CurrentAroundTargetAccumulation", 
+        "#");
+
+        public static Metric<TraceGC> RecordedTcpCount = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.RecordedTcpCount;
+        },
+        "RecordedTcpCount", 
+        "#");
+
+        public static Metric<TraceGC> RecordedTcpSlope = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.RecordedTcpSlope;
+        },
+        "RecordedTcpSlope", 
+        "#");
+
+        public static Metric<TraceGC> NumGcsSinceLastChange = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.NumGcsSinceLastChange;
+        },
+        "NumGcsSinceLastChange", 
+        "#");
+
+        public static Metric<TraceGC> AggFactor = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.AggFactor;
+        },
+        "AggFactor", 
+        "#");
+
+        public static Metric<TraceGC> ChangeDecision = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.ChangeDecision;
+        },
+        "ChangeDecision", 
+        "#");
+
+        public static Metric<TraceGC> AdjReason = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.AdjReason;
+        },
+        "AdjReason", 
+        "#");
+
+        public static Metric<TraceGC> HcChangeFreqFactor = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.HcChangeFreqFactor;
+        },
+        "HcChangeFreqFactor", 
+        "#");
+
+        public static Metric<TraceGC> HcFreqReason = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.HcFreqReason;
+        },
+        "HcFreqReason", 
+        "#");
+
+        public static Metric<TraceGC> AdjMetric = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationTuning?.AdjMetric;
+        },
+        "AdjMetric", 
+        "#");
+
+        public static Metric<TraceGC> ElapsedTimeBetweenGCs = new(gc => 
+        {
+            return gc.DynamicEvents().SizeAdaptationSample?.ElapsedTimeBetweenGCs;
+        },
+        "ElapsedTimeBetweenGCs", 
+        "msec");
```

Comment:
I think `ms` is already used for this unit.  They need to match for the axis to be shared.

All of the names/titles should be checked against the others for consistency and that each has enough context.  Possibly these should be prefixed/described in order to scope them to datas.  What is `AdjMetric`? etc.